### PR TITLE
Added test for ast-utils/assertRoot.ts

### PR DESCRIPTION
**Pull request by jer3m01**
## Summary
Part of #1023 

Adds test for ast-utils/assertRoot.ts

## Test Plan

`rome test` passes all tests.

### CODE REVIEW

File: internal/ast-utils/assertRoot.test.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	path: "unknown",
+	input: "<div></div>",
+});
+
+const nodes = [jsNode, cssNode, commitNode, mdRoot, htmlNode];
+
+test(
+	"nodes",
+	async (t) => {
+		nodes.map((node) => {
```
Comment (ematipico) [inline internal/ast-utils/assertRoot.test.ts:45]: These snapshots will be/already are covered insider the parsers, so I'd say we can remove them

File: internal/ast-utils/assertRoot.test.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+			t.is(assertRoot(node), node);
+		});
+	},
+);
+
+test(
+	"throws if not a root node",
+	async (t) => {
+		nodes.map((node: AnyRoot) => {
```
Comment (ematipico) [inline internal/ast-utils/assertRoot.test.ts:63]: There isn't need of a loop here. (and `.map` leaks memory)
Just pick `commitRoot.body[0]` and you're fine

**Comment by sebmck** [comment]
This is a pretty basic function so I'm not sure we even need tests for it. It's just refining the `type` field. I think the additional snapshots and having to parse each is unnecessary. If we really want to test this then we should just inline the AST. ie. construct an empty root object.

**Comment by sebmck** [comment]
Going to close this out for the previously mentioned reason. Happy to clarify any questions or concerns in the linked issue. Thank you @jer3m01!

**Pull request closed without merging**